yarn.lock

```diff
@@ -4,17 +4,17 @@
 
 "@types/highlight.js@^9.7.0":
   version "9.12.4"
-  resolved "https://registry.yarnpkg.com/@types/highlight.js/-/highlight.js-9.12.4.tgz#8c3496bd1b50cc04aeefd691140aa571d4dbfa34"
+  resolved "https://registry.npmjs.org/@types/highlight.js/-/highlight.js-9.12.4.tgz"
```

Comment:
I'm not much of a node/yarn/etc expert but shouldn't the lock files be excluded from repos? (yarn.lock, package-lock.json, etc)

Reply:
AFAIK it should be committed because they contain the exact version used: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5